data/hai/hai reveal 4 middle.txt

```diff
@@ -1417,25 +1417,59 @@ mission "Hai Reveal [A14] A Time of Trouble"
 		log `The Hai have threatened the Republic if suitable action isn't taken. Required to escort civilians out of the possible danger zone.`
 		conversation
 			`Upon docking at <planet>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`
-			`	Sayari is intercepted by a Hai functionary and led away as you reach Xilin and the other diplomats, also joined by Danforth. Xilin looks at you all. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. The Navy has blockaded the wormhole to the best of its ability, but the reality is that the Navy is already heavily committed, and barely look to be a match for the Hai fleet numbers. The Hai, on the other hand, do not yet appear to be committed heavily." He looks at Alondo briefly. "We had the unstated privilege of observing the Hai security fleet left behind on Everhope in addition to what we saw beyond the wormhole on our way in."`
+			`	As you reach Xilin and the other diplomats, who is also joined by Danforth, Sayari is intercepted by a Hai functionary and led away. Xilin looks at you all with a grave face, and begins. "So. The Hai have broadcast the threat that if the Republic does not treat with them seriously, the united Hai will move in without Republic approval to search for their people. We're at a political impasse now, as the cabinet is blocking any progress.`
+			`	"The Hai have demanded we cut through the red tape by having the heads of state meet. They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`
```

Comment:
```suggestion
			`	"The Hai have demanded we cut through the red tape by having the heads of state meet." He turns towards you briefly. "They insist on you as a mediator. I think behind-the-scenes discussions have reached the point where we can proceed. We've gathered to brief you before the meeting."`
```
It could be done this way too.

Reply:
I used this option, but having "brief" twice in one sentence is confusing, so I also changed the last sentence to "We've gathered to prepare you for the meeting."

Reply:
That is better, just needs one more thing.